bazel/oci/install_bazel.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+curl \
+    --fail \
+    --location \
+    --remote-name \
+    "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64"
+
+curl \
+    --fail \
+    --location \
+    "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64.sha256" \
+    | sha256sum --check
+
+mv "bazel-${BAZEL_VERSION}-linux-x86_64" bazel
```

Comment:
Can we make this architecture aware to support containers that work on arm64, too?